Library/Classes/After the End/Nomad.gct

```diff
@@ -27,32 +27,32 @@
 					</attribute_bonus>
 				</advantage>
 				<advantage version="3">
-					<name>Increased Health</name>
-					<type>Physical</type>
+					<name>Increased Intelligence</name>
+					<type>Mental</type>
 					<levels>2</levels>
-					<points_per_level>10</points_per_level>
-					<reference>B14</reference>
+					<points_per_level>20</points_per_level>
+					<reference>B15</reference>
 					<categories>
 						<category>Advantage</category>
 						<category>Attribute</category>
 					</categories>
 					<attribute_bonus>
-						<attribute>ht</attribute>
+						<attribute>iq</attribute>
 						<amount per_level="yes">1</amount>
 					</attribute_bonus>
 				</advantage>
 				<advantage version="3">
-					<name>Increased Intelligence</name>
-					<type>Mental</type>
+					<name>Increased Health</name>
+					<type>Physical</type>
 					<levels>1</levels>
-					<points_per_level>20</points_per_level>
-					<reference>B15</reference>
+					<points_per_level>10</points_per_level>
+					<reference>B14</reference>
 					<categories>
 						<category>Advantage</category>
 						<category>Attribute</category>
 					</categories>
 					<attribute_bonus>
-						<attribute>iq</attribute>
+						<attribute>ht</attribute>
```

Comment:
All of the changes to this file seem to be superfluous -- looks like the order of the Increased Health and Increased Intelligence were swapped. Unless there is a good reason for this being necessary, please eliminate these changes.

Reply:
Sorry for the confusion with the order, what this change actually does is changing HT+2, IQ+1 to IQ+2, HT+1. I changed the order to match the book and didn't realize that would make the diff look confusing.

Reply:
Ah... yeah... just changing the level’s value would have been clearer.